tests/kernel/mem_protect/userspace/src/main.c

```diff
@@ -829,6 +829,12 @@ void test_object_recycle(void)
 {
 	struct z_object *ko;
 	int perms_count = 0;
+	int dummy = 0;
+
+	/* Validate recycle objects with NULL parameter */
+	ko = z_object_find(&dummy);
+	zassert_true(ko == NULL, "not an invalid object");
+	z_object_recycle(&dummy);
```

Comment:
Suggests we add a perms_count check here.

Reply:
@enjiamai , since `ko` here is NULL, so we can not check  perms_count here.